templates/blog-post.html

```diff
@@ -27,5 +40,30 @@ <h2 class="mb-0">{{ this.title }}</h2>
       </p>
     </div>
   </div>
+{% if this.series|length %}
+<table class="table table-sm table-striped mt-3" id="series">
```

Comment:
I think this table is a bit too large and I don't think we want to use a black background for the header here, it draws too much attention compared to the rest of the blog post. Please make the style really simple and compact (see the table of contents styling on this page: https://opensource.creativecommons.org/contributing-code/github-repo-guidelines/ for an example).